autopts/wid/mics.py

```diff
@@ -0,0 +1,103 @@
+#
+# auto-pts - The Bluetooth PTS Automation Framework
+#
+# Copyright (c) 2023, Codecoup.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms and conditions of the GNU General Public License,
+# version 2, as published by the Free Software Foundation.
+#
+# This program is distributed in the hope it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+
+import logging
+import sys
+
+from autopts.ptsprojects.stack import get_stack
+from autopts.pybtp import btp
+from autopts.pybtp import defs
+from autopts.pybtp.types import WIDParams
+
+log = logging.debug
+mute_state = None
+
+
+def mics_wid_hdl(wid, description, test_case_name, logs=True):
+    if logs:
+        log(f'{mics_wid_hdl.__name__}, {wid}, {description}, {test_case_name}')
+    module = sys.modules[__name__]
+
+    try:
+        handler = getattr(module, f'hdl_wid_{wid}')
+        return handler(WIDParams(wid, description, test_case_name))
+    except AttributeError as e:
+        logging.exception(e)
```

Comment:
I did not notice in the MICP PR, but you used there and here the old convention (see other files in autopts/wid for reference). Please use the new one:
```suggestion
def mics_wid_hdl(wid, description, test_case_name):
    log(f'{mics_wid_hdl.__name__}, {wid}, {description}, {test_case_name}')
    return generic_wid_hdl(wid, description, test_case_name, [__name__])
```